wire-ios/Wire-iOS/Generated/Strings+Generated.swift

```diff
@@ -2099,8 +2099,8 @@ internal enum L10n {
           internal static let cancel = L10n.tr("Localizable", "conversation.create.mls.cancel", fallback: "Cancel")
           /// Select Protocol
           internal static let pickerTitle = L10n.tr("Localizable", "conversation.create.mls.picker_title", fallback: "Select Protocol")
-          /// Select MLS to create a MLS conversation (Beta functionality, use at your own risk).
-          internal static let subtitle = L10n.tr("Localizable", "conversation.create.mls.subtitle", fallback: "Select MLS to create a MLS conversation (Beta functionality, use at your own risk).")
+          /// Select MLS to create a group using Messaging Layer Security protocol (beta version).
```

Comment:
I think you should change strings in the main file, not a generated one.

Reply:
It wasn't my change tbh. I think someone just pushed changes without triggering script for generating strings but i left it because at some point it has to be updated

Reply:
@KaterinaWire it's from develop just ignore, I didn't commit it